engine/swarm/admin_guide.md

```diff
@@ -251,8 +237,51 @@ Before you forcefully remove a manager node, you must first demote it to the
 worker role. Make sure that you always have an odd number of manager nodes if
 you demote or remove a manager
 
+## Back up the swarm
+
+Docker manager nodes store the swarm state and manager logs in the
+`/var/lib/docker/swarm/raft` directory.
+
+Back up the `raft` data directory often so that you can use it in case of
+[disaster recovery](#recover-from-disaster). To restore, see
+[Restore from a backup](#restore-from-a-backup).
+
+### Things to know before backing up
+
+#### Hot or cold backups
+
+To do a _cold backup_, completely shut down the swarm before backing it up.
+Generally, _hot backups_ are sufficient. However, if any files are being changed
+while the copy is occurring, you may lose Some data. Be sure to test backup
+and recovery  thoroughly before settling on a backup strategy.
```

Comment:
I think I've changed my mind about this. Let's recommend that people do cold backups. It's not very reassuring to say that hot backups will work, except when they don't work.